package.json

```diff
@@ -37,6 +37,7 @@
     "@babel/core": "^7.8.4",
     "@babel/runtime": "^7.8.4",
     "@react-native-community/eslint-config": "^1.1.0",
+    "@types/htmlparser2": "^3.10.1",
```

Comment:
You would need to add htmlparser2 types to dependencies instead of devDependencies. Otherwise, typescript users will encounter type errors when importing this library (because you depend on them in index.d.ts).

Reply:
Alright! thanks for the review, i'll sort out the issues.

Reply:
Thanks! If you struggle with the test or have any question, ping me on our Discord :-)

Reply:
Here is a link to our [Discord #contributing channel](https://discord.gg/5eVdDp)